src/main/resources/templates/plugins/task/editArtifactoryGradleBuilder.ftl

```diff
@@ -75,6 +75,8 @@ listKey='repoKey' listValue='repoKey' toggle='true'/]
 
 [@ui.bambooSection dependsOn='publishArtifacts' showOn=true]
 
+    [@ww.textfield labelKey='artifactory.task.gradle.publishForkCount' name='builder.artifactoryGradleBuilder.publishForkCount' value='${publishForkCount}'/]
```

Comment:
What do you say we change the text field into a select-box with a few values to choose from? This way, we can be sure that the user input is numeric.

Reply:
I thought about that too initially :)
Done